.vscode/launch.json

```diff
@@ -15,7 +15,7 @@
             "sourceMaps": true,
             "env": {
                 "NODE_ENV": "local",
-                "IotHubConnectionString": "",
+                "IotHubEventHubConnectionString": "",
```

Comment:
The latest event hub npm package requires an event hub connection string, or the event hub name. Rather than add a 3rd variable, I chose to change the connection string type. To make sure people don't put in the wrong connection string, I changed the name of this variable to be explicit.